app/Http/Requests/UserUpdateRequest.php

```diff
@@ -24,16 +24,21 @@ public function authorize()
      */
     public function rules()
     {
-        return [
+        $rules = [
             'name' => 'required|min:5|max:50',
             'email' => [
                 'required',
                 'max:200',
                 Rule::unique('users')->ignore($this->id),
             ],
-            'password' => 'required|min:8|max:50',
             'avatar' => 'image|max:5120',
         ];
+        if ($this->password != '') {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/Http/Requests/UserUpdateRequest.php</br>**Line**: 36</br>Blank line found at start of control structure